CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.20.0] - XXX. XX, XXXX
+## [0.20.0] - Jun. 03, 2025
 
```

Comment:
Would it make sense to add also some highlights for the relese?

Reply:
I completely forgot, it would yes, good point

Reply:
I'll make a separate PR into master with it and then backport that separately